src/fitnesse/slim/converters/LongConverter.java

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2003-2009 by Object Mentor, Inc. All rights reserved.
```

Comment:
Why this copyright message?

Reply:
I believe this copyright was in the [IntConverter.java](https://github.com/unclebob/fitnesse/blob/30b496e330add41ab36b7fa04b21f1a6e8fefecd/src/fitnesse/slim/converters/IntConverter.java), so it was inherited by copy here